app/services/open_library_service.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+class OpenLibraryService
+  require 'HTTParty'
+
+  BASE_URI = Rails.application.credentials.config[:api_open_library][:base_uri]
+
+  def initialize(isbn)
+    @isbn = isbn
+  end
+
+  def book_info
+    response = request_to_open_library
+    if response.empty?
+      { status?: :bad_request, result: { message: 'Book not found' } }
```

Comment:
@blacksam07 Are we gonna use i18n or not yet?

Reply:
Also, I'm not so sure about responding with "bad_request" when the book was not found. Bad request implies an improper use of the endpoint. Not finding a resource implies proper use but the resource doesn't exist

Reply:
@redwarewolf yes I had not seen `bad_request`, not the best response code @maxy96 check this page https://httpstatuses.com/

Reply:
@redwarewolf not for now

Reply:
So the status should be 200?

Reply:
@maxy96 in that not found case you should 404  or :not_found

Reply:
ok